apps/yarn.lock

```diff
@@ -1738,6 +1744,14 @@ chalk@^0.5.0, chalk@^0.5.1, chalk@~0.5.1:
     strip-ansi "^0.3.0"
     supports-color "^0.2.0"
 
+chalk@^2.0.1:
```

Comment:
Is yarn.lock properly updated?  I'd expect `chalk@latest` to resolve here but it's still resolving to 1.1.3 (about 20 lines up).